packages/salesforcedx-vscode-core/src/commands/forceSourceDeployManifest.ts

```diff
@@ -49,10 +53,23 @@ export async function forceSourceDeployManifest(manifestUri: vscode.Uri) {
       return;
     }
   }
+
+  const messages: ConflictDetectionMessages = {
+    warningMessageKey: 'conflict_detect_conflicts_during_deploy',
+    commandHint: input => {
+      return new SfdxCommandBuilder()
+        .withArg('force:source:deploy')
+        .withFlag('--manifest', input)
```

Comment:
Need to add `.withLogName('<metric key>')` for this command to generate telemetry.

Reply:
This builder is used to generated output to the console, so telemetry is not necessary.

Reply:
Got it 👍